README.md

```diff
@@ -45,9 +51,15 @@
 
 ### Key Features <a name="key-features"></a>
 
-- **Uses HtmlWebpackPlugin to automatically create the index.html file in the /dist directory**
-- **Style-loader and css-loader added to module configuration**
-- **Webpack dev server set up**
+- **Easy navigation**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- - [Live Demo Link](https://mutugi-c.github.io/To_do_list/dist/)
```

Comment:
- [ ] @mutugi-c great work on your documentation, I loved it, however, am getting the error below when I try to access your live demo, kindly consider taking a look at the live demo link to make sure it works correctly
![image](https://user-images.githubusercontent.com/61408860/236293570-2d8da973-a81f-4628-b6af-cce9ff0ee225.png)